akka-javasdk/src/main/scala/akka/javasdk/impl/SdkRunner.scala

```diff
@@ -192,16 +191,24 @@ class SdkRunner private (dependencyProvider: Option[DependencyProvider], disable
             serviceName = applicationConf.getString("akka.javasdk.dev-mode.service-name"),
             eventingSupport = extractBrokerConfig(applicationConf.getConfig("akka.javasdk.dev-mode.eventing")),
             mockedEventing = SpiMockedEventingSettings.empty,
-            testMode = false))
+            testSetting = new SpiTestSettings(testMode = false, debugTracing = false),
+            selfServiceName = None))
```

Comment:
@johanandren can you confirm is this is the right selfServiceName? Is that set from testkit somehow, or how is that used?

Reply:
@pvlugter is debugTracing always false in SpiTestSettings or how is that configured?

Reply:
That debug tracing is for accessing tracing in tests, that @aludwiko worked on:

- https://github.com/akka/akka-sdk/pull/965